dlp/snippets/redact/redact.go

```diff
@@ -61,7 +61,7 @@ func redactImage(w io.Writer, projectID string, infoTypeNames []string, bytesTyp
 	// Read the input file.
 	b, err := ioutil.ReadFile(inputPath)
 	if err != nil {
-		return fmt.Errorf("ioutil.ReadFile: %w", err)
+		return fmt.Errorf("ioutil.ReadFile: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
fixed it.